modules/ROOT/pages/partner-manager-identifiers.adoc

```diff
@@ -80,34 +80,34 @@ Use the following values to configure the X12 GS identifier type:
+The receiving partner information about the application message from the backend application might not necessarily be the same as the name of the partner in Partner Manager. This scenario is similar to how ISA and GS identifiers on an inbound EDI message might not be the same as the profilename of the partner.
 
 Partner Manager’s outbound message processing framework makes a dynamic lookup to determine the receiving partner based on the content in the application message received from the backend systems.
 
 === Example
 
-In this example, an outbound purchase order XML message from your ERP system to your trading partner Mythical Supplier has the following content in the payload:
+In this example, an outbound purchase order XML message from your ERP system to your partner Mythical Supplier has the following content in the payload:
 
 `<VendorName>MYTHICAL SUPPLIER, LLC</VendorName>`
 
-An outbound payment remittance advice JSON message from your Financials system to the same trading partner contains the following content in the payload:
+An outbound payment remittance advice JSON message from your Financials system to the same partner contains the following content in the payload:
 
 `"PayeeName": "MYTHICAL ENTERPRISES"`
 
-In order for Partner Manager to route these outbound transactions to the appropriate message flows for Mythical Supplier, you must add these values as Partner Reference IDs from the partner’s profile page by <<create-identifier,creating a new identifier>> and selecting the type as *Reference ID*.
+For Partner Manager to route these outbound transactions to the appropriate message flows for Mythical Supplier, you must add these values as Partner Reference IDs from the partner’s profile page. To do this <<create-identifier,create a new identifier>> and select the type as *Reference ID*.
 
-If your backend application sends outbound transactions to trading partners on behalf of more than one internal line of business, thus requiring you to set different X12 ISA and X12 GS sender identifiers on the transformed outbound data (depending on which LoB is sending the transaction), you can configure partner reference identifiers on your host profile, with the value in the application message’s payload that uniquely identifies the sending LoB.
+If your backend application sends outbound transactions to partners on behalf of more than one internal line of business, thus requiring you to set different X12 ISA and X12 GS sender identifiers on the transformed outbound data (depending on which LoB is sending the transaction), you can configure partner reference identifiers on your host profile, with the value in the application message’s payload that uniquely identifies the sending LoB.
```

Comment:
Should backend be "back-end" in this case, because it's an adjective, not a noun?

Reply:
This sentence is super long. Is there any way to break it up/simplify?

Reply:
OK to the first comment. Not sure how to break this up.